lib/lookup.json

```diff
@@ -514,35 +425,13 @@
     "stripAnsi": true,
     "maintainers": "mcollina"
   },
-  "torrent-stream": {
-    "prefix": "v",
-    "maintainers": "mafintosh"
-  },
-  "tough-cookie": {
-    "maintainers": ["awaterma", "colincasey", "ruoho", "wjharney"]
-  },
   "uglify-js": {
     "prefix": "v",
     "flaky": ["ppc", "darwin"],
     "maintainers": "mishoo",
     "skip": ["12", true],
     "comment": "Tests timeout"
   },
-  "underscore": {
-    "flaky": ["aix", "s390"],
-    "skip": "win32",
-    "maintainers": "jashkenas",
-    "ignoreGitHead": true
-  },
-  "undici": {
-    "prefix": "v",
-    "maintainers": ["mcollina", "ronag"]
-  },
-  "uuid": {
```

Comment:
I just release `uuid@v9.0.1` which should fix the test failure. Can you add it back in?

Reply:
I'll add it back in and re-run CITGM. 🤞